apps/ui/src/components/Layout/App.vue

```diff
@@ -67,6 +72,25 @@ const hasTopNav = computed(() => {
   return 'space-editor' !== String(route.matched[1]?.name);
 });
 
+const skinVariables = computed(() => {
+  if (!whiteLabelSpace.value?.additionalRawData?.skinSettings) return {};
+
+  const colors = clone(whiteLabelSpace.value?.additionalRawData?.skinSettings);
+
+  const result = Object.entries(colors).reduce((acc, [colorName, hex]) => {
+    if (!hex || !colorName.includes('_color')) return acc;
+
+    const rgb = hexToRgb(hex.slice(1));
```

Comment:
Maybe better to store RBG value at DB level instead?

Reply:
Storing them as hex is more concise and easier to read.